README.md

```diff
@@ -113,6 +113,33 @@ RUN dpkg --add-architecture arm64 && \
 $ docker build -t my/image:tag path/to/where/the/Dockerfile/resides
 ```
 
+### Passing environment variables into the build environemnt
+
+By default, `cross` does not pass any environment variables into the build
+environment from the calling shell. Sometimes this is really useful, for
```

Comment:
Also, given what came up in japaric/trust#73, I suggest it also be made clear that Mac doesn't use docker and so all environment variables are automatically "passed through" and this whitelist is only used for docker-targets (currently just Linux).